aries-site/src/pages/design-tokens/layout-and-spacing.mdx

```diff
@@ -5,101 +5,141 @@ import { ContentPane, BestPracticeGroup, Example } from '../../layouts';
-
-The sizing scale consists of a primary scale based on the doubling/halving of a 24px base unit and a supplementary scale doubling/halving a 16px base unit.
-
-The resulting scale steps: 1, 2, 3, 4, 6, 8, 12, 16, 24, 32, 48, 64, 96, 128, 192, 256, 384, 512, 768, 1024, 1536.
-
-[Spacing](#spacing), [container](#container), [radius](#radius), and [borderWidth](#border-width) tokens are defined using subsets of this scale.
 
 ## Spacing
 
-Spacing uses proximity principles to define relationships between content and create hierarchy within a page. Spacing design tokens define the padding within, margin around, or gap between content.
-
-Spacing guides users’ eyes through the page, helping them understand the relationships between different elements. The HPE Design System minimizes use of borders to separate content. Instead spacing, typography, and background color are combined to create visual hierarchy
-
-There are 12 spacing tokens to provide flexibility to cater to various design needs ranging from dense data-rich experiences to more spacious layouts.
+Spacing tokens control `pad`, `gap`, and `margin` through `edgeSize`.
 
 <ContentPane>
   <DesignTokensTable active="semantic.spacing" toolbar />
 </ContentPane>
 
-### Spacing best practices
-
-<BestPracticeGroup>
-  <Example
-    height="none"
-    bestPractice={{
-      type: 'do',
-      message:
-        'Use spacing design tokens to style compositional or layout components like cards, layers, and grid spacing.',
-    }}
-    width="100%"
-  >
-    <SpacingBestPractices />
-  </Example>
-  <Example
-    height="none"
-    bestPractice={{
-      type: 'dont',
-      message: `Use spacing design tokens to style element-level components. Instead use component-specific tokens or element tokens which will ensure correct sizing of elements across screen sizes.`,
-    }}
-  >
-    <SpacingBestPractices bestPractice={false} />
-  </Example>
-</BestPracticeGroup>
 
 ## Containers
 
-Containers define the minimum or maximum width or height for content areas.
-
-Container tokens are particularly useful when defining grid layouts. In other cases, consider the approach of [content-driven layouts](/templates/content-layouts#content-driven-layouts) to avoid artificially restricting content.
-
-There are 11 container tokens to provide flexibility to cater to various design needs.
+Container tokens define `width` and `height` dimensions for components such as [Box](/components/box).
 
 <ContentPane>
   <DesignTokensTable active="semantic.container" />
 </ContentPane>
 
-## Radius
 
-Radius tokens define the rounding of corners on elements. Radius can be used as a tool to further emphasize the shape or relationship between certain elements.
+## Radius
 
-Larger elements, like page sections, may have a larger radius while elements nested within may have smaller radii.
+Radius tokens define corner rounding for elements.
```

Comment:
We should be consistent with the tokens terminology we use. I believe there was a conscious choice to use the term "design tokens" for clarity as the generic "token" is applied to many things (e.g. access tokens).

```suggestion
Radius design tokens define corner rounding for elements.
```